t/destructive/peers_test.go

```diff
@@ -0,0 +1,112 @@
+package destructive
+
+import (
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/status-im/status-go/geth/api"
+	"github.com/status-im/status-go/t/e2e"
+	. "github.com/status-im/status-go/t/utils"
+
+	"github.com/ethereum/go-ethereum/p2p"
+	"github.com/ethereum/go-ethereum/p2p/discover"
+	"github.com/stretchr/testify/suite"
+)
+
+const (
+	defaultTimeout = 30 * time.Second
+)
+
+func TestPeersSuiteLinkUpDown(t *testing.T) {
+	suite.Run(t, &PeersTestSuite{tester: new(NetworkConnectionTester)})
+}
+
+type PeersTestSuite struct {
+	suite.Suite
+
+	backend *api.StatusBackend
+	tester  *NetworkConnectionTester
+}
+
+func (s *PeersTestSuite) SetupTest() {
+	netid := GetNetworkID()
+	s.Require().NotEqual(0, netid, "test suppose to work only on public network")
```

Comment:
I don't really sure how `GetNetworkID() != 0` would make sure that a public network is used. Our private network uses `777`.

Reply:
thanks, missed that)